lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
Actually @jcmeloni if we can throw that in as a gate, I can say in this context it is no longer "willy nilly." In that MHV does want this data accurately represented. They told me that the only boolean value that is important is the "is_veteran" one. They depend on that one in particular and we should make efforts to provide it accurately if possible. Are you aware of means whereby we can determine this currently? 

I actually agree with Patrick and feel equally uncomfortable hardcoding those boolean values, but  I see no real alternative presently.